domain-cc/cc-app/build.gradle

```
@@ -9,16 +9,27 @@ python {
   installVirtualenv = true
 }
 
+def devrequirements = tasks.register("devrequirements", PythonTask) {
+  module = 'pip'
+  command = "install -r src/dev-requirements.txt"
+}
+
```

IMO: This looks cleaner
```suggestion
tasks.register("devrequirements", PythonTask) {
  module = 'pip'
  command = "install -r src/dev-requirements.txt"
}

```